rllib/algorithms/algorithm_config.py

```diff
@@ -197,6 +197,9 @@ def __init__(self, algo_class=None):
         self.min_train_timesteps_per_iteration = 0
         self.min_sample_timesteps_per_iteration = 0
 
+        # `self.checkpointing()`
+        self.checkpoints_contain_native_model_files = False
```

Comment:
New config option to store NN models as native torch/keras models inside a policy's checkpoint directory (under sub-dir: "model").